README.md

```diff
@@ -1,27 +1,34 @@
 # Role Name
 
-A brief description of the role goes here.
+This role installs and configures a redis server. It allows to set all configuration variables supported by redis.
 
 
 ## Requirements
 
-Any pre-requisites that may not be covered by Ansible itself or the role should be mentioned here.
-For instance, if the role uses the EC2 module or depends on other Ansible roles, it may be a good idea to mention in this section that the boto package is required.
+An dpkg- or pacman-based Linux distribution.
 
 
 ## Role Variables
 
-A description of the settable variables for this role should go here, including any variables that are in defaults/main.yml, vars/main.yml, and any variables that can/should be set via parameters to the role.
-Any variables that are read from other roles and/or the global scope (ie. hostvars, group vars, etc.) should be mentioned here as well.
-
+This role seriously has a ton of variables. Instead of copying the defaults file here, look it up there. All variables from redis.conf are called exactly like they are called in the file but with redis_ prepended.
```

Comment:
@SlothOfAnarchy What do you think?

Reply:
Does every single var have a default value? Is nothing mandatory?
I guess the user has to go through the template file and read the comments there to understand and configure...

Reply:
yes everthing is default you can download and run redis without configuring anything